### ISSUE TITLE
searchstring with accentuated characters

### ISSUE THREAD

**Issue by Coroico** [Coroico/AdvSearch#9]
It doesn't be possible to do a search with a searchterm with accentuated characters.

"über" wich means "over" or "ähnlich" wich means "similar". 
Even words with accent in french are not possible like "téléphone".

**Comment by Coroico** [comment]
To fix this issue, line 183 of the file core/components/advsearch/model/advsearch/classes/advsearch.class.php replace:

```
        $searchQuery = Zend_Search_Lucene_Search_QueryParser::parse($searchString);
```

by:

```
        $searchQuery = Zend_Search_Lucene_Search_QueryParser::parse($searchString, $this->config['charset']);
```

**Comment by Coroico** [comment]
Fixed with dc6e5a8591 commit
